cli/cmd/component-apply.go

```diff
@@ -61,12 +61,18 @@ func runApply(cmd *cobra.Command, args []string) {
 		log.SetLevel(log.DebugLevel)
 	}
 
+	if err := componentApply(contextLogger, args); err != nil {
+		contextLogger.Fatalf("Applying components failed: %v", err)
+	}
+}
+
+func componentApply(contextLogger *log.Entry, componentsList []string) error {
```

Comment:
I have mixed feelings about this function.

We now have `componentApply` and `applyComponents`. It's little confusing don't you think? Can you please change name of function or add comment

Reply:
cc @invidian

Reply:
I added comments for both to make it easier to understand. TL; DR `componentApply` represents `lokoctl component apply` subcommand functionality, but separated from CLI dependencies/concepts, hence the name.

Please note, that this is not the final version of this code, there is much much more which needs to be changed to get it to well-readable state, but I'm doing it all in small steps to make it easier to review.